apps/deploy-web/src/components/usage/UsageLayout.tsx

```diff
@@ -0,0 +1,54 @@
+"use client";
+
+import type { ReactNode } from "react";
+import React from "react";
+import { ErrorBoundary } from "react-error-boundary";
+import { ErrorFallback, Tabs, TabsList, TabsTrigger } from "@akashnetwork/ui/components";
+import { cn } from "@akashnetwork/ui/utils";
+import { useRouter } from "next/navigation";
+
+import { Title } from "@src/components/shared/Title";
+import { UrlService } from "@src/utils/urlUtils";
+
+export enum UsageTabs {
+  BILLING = "BILLING",
+  USAGE = "USAGE"
+}
+
+type Props = {
+  page: UsageTabs;
+  children?: ReactNode;
+};
+
+export const UsageLayout: React.FunctionComponent<Props> = ({ children, page }) => {
+  const router = useRouter();
+
+  const handleTabChange = (newValue: string) => {
```

Comment:
methods should state what they do, not when they're called
```suggestion
  const navigateToTab = (newValue: string) => {
```

Reply:
After removing the Billing page, this method doesn't exist anymore